File: 백준/Bronze/2753. 윤년/윤년.js
const input = require("fs")
  .readFileSync(
    process.platform === "linux"
      ? "/dev/stdin"
      : require("path").resolve(__dirname, "./testcase.txt")
  )
  .toString()
  .trim()
  .split("\n");

N = input.map(Number);
N = N.join("");


function leapYear(year) {
    if(year % 4 === 0 && (year % 100 !== 0 || year % 400 === 0)) {
        return 1
    }
    else {
        return 0
    }
}


console.log(leapYear(N))

File: 백준/Bronze/2753. 윤년/윤년.py
a = int(input())

1 <= a <= 4000

x = (a % 4 == 0 and a % 100 != 0)
y = (a % 400 == 0)

if x:
    print("1")
elif y:
    print("1")
else:
    print("0")